pkg/controller/master/controller.go

```diff
@@ -13,6 +11,7 @@ import (
 	"github.com/rancher/harvester/pkg/controller/master/user"
 	"github.com/rancher/harvester/pkg/controller/master/virtualmachine"
 	"github.com/rancher/harvester/pkg/indexeres"
+	"github.com/rancher/steve/pkg/server"
```

Comment:
this was done in purpose by separating the 3rd party package and self package.

Reply:
fixed